app/error/errors.js

```diff
@@ -134,3 +134,9 @@ export class ErrorUnexpected extends HttpError {
     res.status(500).json({ error: "An unexpected error occurred.", message: this.message });
   }
 }
+
+export class ErrorUniqueType extends HttpError {
+  throwHttp(req, res) {
+    res.status(500).json({ error: "Entry has duplicated unique entry."})
+  }
+}
```

Comment:
This new error is named rather ambiguously. `UniqueType` makes me think of some Typescript implementation, or something completely semantically different that a duplicate entry in the database. 

I would move this error to under the `409 Conflict` errors, and call it something like `ErrorDuplicateKey` or something semantically clearer that describes what the error is.